labs/lab1/chapter1.md

```diff
@@ -34,40 +34,40 @@ Now check on the status of docker.
 $ systemctl status docker
 ```
 
-## docker Help
+## podman Help
 
-Now that we see how the docker startup process works, we should make sure we know how to get help when we need it.  Run the following commands to get familiar with what is included in the docker package as well as what is provided in the man pages. Spend some time exploring here. 
+Now that we see how the podman startup process works, we should make sure we know how to get help when we need it.  Run the following commands to get familiar with what is included in the podman package as well as what is provided in the man pages. Spend some time exploring here. 
 
 Check out the executables provided:
 ```bash
-$ rpm -ql docker | grep bin
+$ rpm -ql podman | grep bin
 ```
 
 Check out the configuration files that are provided:
 ```bash
-$ rpm -qc docker
+$ rpm -qc podman
 ```
 
 Check out the documentation that is provided:
 ```bash
-$ rpm -qd docker
+$ rpm -qd podman
 ```
 
-Run `docker {help,info}` to check out the storage configuration and how to find more information. 
+Run `podman {help,info}` to check out the storage configuration and how to find more information. 
 ```bash
-$ docker --help
-$ docker run --help
-$ docker info
+$ sudo podman --help
+$ sudo podman run --help
```

Comment:
let's take the sudo off these first two commands. we don't need root access to run `--help`